packages/battery_plus/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 1.1.2
 
+* Update battery_plus to 4.0.1.
```

Comment:
This is wrong. battery_plus_tizen 1.1.2 had already been released before this PR was merged.